src/utils/auth.ts

```diff
@@ -2,6 +2,22 @@ import { SiweMessage } from 'siwe'
 import { User } from '@/types/user/User'
 import { axiosInstance } from './axiosInstance'
 
+axiosInstance.interceptors.response.use(
+  function (response) {
+    return response
+  },
+  function (error) {
+    if (error.response && error.response.status === 401) {
+      window.localStorage.removeItem('auth')
```

Comment:
I think you can reuse the `logout` function here. They seem to be doing the same thing.